std_msgs/Bool red
int32[] red_box
std_msgs/Bool green 
int32[] green_box
std_msgs/Bool blue
int32[] blue_box

